gadget/gadget_test.go

```diff
@@ -2262,6 +2359,12 @@ func (s *gadgetYamlTestSuite) TestLaidOutVolumesFromGadgetUC20Happy(c *C) {
 	c.Assert(systemLv.LaidOutStructure, HasLen, 6)
 }
 
+func (s *gadgetYamlTestSuite) TestLaidOutVolumesFromGadgetUCHappy(c *C) {
+	s.testLaidOutVolumesFromGadgetUCHappy(c, gadgetYamlUC20PC)
+	// TODO fix after layouts stuff is fixed
+	//s.testLaidOutVolumesFromGadgetUCHappy(c, gadgetYamlMinSizePC)
```

Comment:
is this work in progress?

Reply:
Actually the TODO was already done, I've uncommented now.